src/docs/operator-guides/how-to-format-your-text-in-pages/index.md

```diff
@@ -0,0 +1,302 @@
+the same as what you can see here.
+
+## Highlighting
+
+You can highlight the text with italics, bold or a combination of both.
+You can do so by adding asterisks or underlines to the text. See the
+examples for reference:
+
+```
+This is *italic text* and this is also _italics text_
+This is **bold text** and this is also __bold text__
+This is ***bold and italics text*** and so ___is this text___
+```
+
+This is _italic text_ and this is also _italics text_ \
+This is **bold text** and this is also **bold text** \
+This is **_bold and italics text_** and so **_is this text_**
+
+## Headers
+
+You can add different levels of headers by adding hashes to the
+beginning of the text. The number of hash signs determines the heading
+level:
+
+```
+### h3 Heading
+#### h4 Heading
+##### h5 Heading
+###### h6 Heading
+```
+
+![headers-example](/headers.png)
+
+It is possible to add h1 and h2 headings to the content via markdown.
+You simply need to follow a similar format. We recommend avoiding doing
+it. A page should only have a single h1 header, which can be added
+directly through the Page editor to the first title in your page. The h2
+can then be either the Section or the Block titles.
+
+## Links
+
+You can add links to the text by adding square brackets, followed by the
+link in parenthesis:
+
+```
+Here is [a link to Sharetribe’s site\]\(https://www.sharetribe.com)
+```
+
+Here is [a link to Sharetribe’s site](https://www.sharetribe.com)
+
+You can also turn a URL or an email address into a link by surrounding
+it with `<>`:
+
+```
+<https://www.sharetribe.com> or <hello@sharetribe.com>
+```
+
+<https://www.sharetribe.com> or <hello@sharetribe.com>
+
+## Lists
+
+You can create ordered or unordered lists.
+
+To create unordered lists you can use different symbols: plus signs,
+dashes or asterisks.
+
+```
+- Product       * Product       + Product
+- Service       * Service       + Service
+- Rental        * Rental        + Rental
+```
+
+- Product
+- Service
+- Rental
+
+To create ordered lists you can use numbers. The first number of the
+list will determine the starting number of the rendered version of the
+list. The other numbers do not matter.
+
+```
+1. First item       1. First item
+2. Second item      1. Second item
+3. Third item       1. Third item
+```
+
+1. First item
+2. Second item
+3. Third item
+
+```
+43. First item
+1. Second item
+2. Third item
+```
+
+43. First item
+1. Second item
+1. Third item
+
+You can also have indented lists by adding four spaces before the list
+item
+
+```markdown
+- Product
+- Service
+- Rental
+  - Sauna rental
+  - Home rental
+- Mixed
+```
+
+- Product
+- Service
+- Rental
+  - Sauna rental
+  - Home rental
+- Mixed
+
+---
+
+```markdown
+1. First item
+2. Second item
+3. Third item
+   1. First indented item
+   2. Second indented item
+4. Fourth item
+```
+
+1. First item
+2. Second item
+3. Third item
+   1. First indented item
+   2. Second indented item
+4. Fourth item
+
+## Image
```

Comment:
I feel that what is missing from this section is that the image needs to be hosted somewhere. A non-technical user doesn't really have a way to upload these images anywhere in Console, so they'd have to host the image somewhere else and point to it. Also, I'm wondering: if you show an image that is hosted somewhere else, does the Content Security Policy document issue a warning of some kind? Vesa L or Boyan might know about this.